ctapipe/reco/hillas_intersection.py

```diff
@@ -381,7 +382,8 @@ def reconstruct_tilted(self, hillas_parameters, tel_x, tel_y):
             core uncertainty X
         """
         if len(hillas_parameters) < 2:
-            return None  # Throw away events with < 2 images
+            return (np.nan, np.nan, np.nan, np.nan)  # Throw away events with < 2 images
```

Comment:
Same question as above.

Reply:
same as above, the function returns a 4-tuple that gets unpacked on usage, so None is not a valid return value